src/Ast.ml

```diff
@@ -2216,7 +2216,9 @@ end = struct
       in
       match ambig_prec (sub_ast ~ctx (Exp exp)) with
       | None -> false (* ctx not apply *)
+                
```

Comment:
@gpetiot, I don't understand why we have this newline here..

Reply:
I'm looking into this, but first I'm trying to fix the other comments-dropped issues that arise after the previous commits of this PR.

Reply:
This is fixed, but there are still some other regressions to fix in the testsuite.